.lintstagedrc.js

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
This is the configuration for [`lint-staged`](https://github.com/lint-staged/lint-staged/), so that it runs `prettier` upon commit, only on the committed/modified files. It replaces `pretty-quick`, but since `lint-staged` is a more general tool, this configuration specifies that it should run `prettier`.